src/routes/privacy/+page.svelte

```diff
@@ -5,90 +5,91 @@
+    information. For residents of the European Economic Area (EEA) or the United Kingdom (UK) please also refer to our <a
+      href="/privacy/gdpr">GDPR statement</a> for regulations that apply to you.</p>
 
-  <h2>Table of Contents</h2>
+  <h2>Table of contents</h2>
   <ul>
-    <li><a href="#contact">Contact Details</a></li>
-    <li><a href="#information-collected">Information We Collect</a></li>
-    <li><a href="#why-collect">Why We Collect Your Information</a></li>
-    <li><a href="#lawful-bases">Our Lawful Bases for Processing Your Data</a></li>
-    <li><a href="#retention">Data Retention Period</a></li>
-    <li><a href="#sharing">Data Sharing and Disclosure</a></li>
-    <li><a href="#last-updated">Last Updated</a></li>
-    <br>
-    <li><a href="#rights">Your Rights</a></li>
-    <li><a href="#complaints">File a Complaint About Our Use of Your Data</a></li>
-
+    <li><a href="#contact">Contact details</a></li>
+    <li><a href="#information-collected">Information we collect</a></li>
+    <li><a href="#why-collect">Why we collect your information</a></li>
+    <li><a href="#lawful-bases">Our lawful bases for processing your data</a></li>
+    <li><a href="#sharing">Who we share your data with and why</a></li>
+    <li><a href="#retention">How long we keep your data for</a></li>
+    <li><a href="#last-updated">Last updated</a></li>
+    <li><a href="#rights">Your rights</a></li>
+    <li><a href="#accessibility">Accessibility</a></li
+    <li><a href="#complaints">Complain about our use of your data</a></li>
   </ul>
 
-  <h2 id="contact">Contact Details</h2>
+  <h2 id="contact">Contact details</h2>
   <p>
     If you have any concerns or questions regarding data protection, please reach out to us at <a
-    href="mailto:data@zenithhacks.org">data@zenithhacks.org</a>. We will respond to your inquiries as soon as possible.
+    href="mailto:data@zenithhacks.org">data@zenithhacks.org</a>. We'll respond to you as soon as possible.
     Please ensure that all emails adhere to our Code of Conduct.
   </p>
 
-  <h2 id="information-collected">Information We Collect</h2>
+  <h2 id="information-collected">Information we collect</h2>
   <p>
-    When you opt in to receive notifications via email using the form on our website, we collect, store, and process the
+    When you opt in to receive a notification via email using the form on our website, we collect, store, and process the
     following information:
   </p>
   <ul>
     <li><b>Email address:</b> Used to notify you when sign-ups open.</li>
-    <li><b>IP address:</b> For security purposes and spam prevention.</li>
-    <li><b>Approximate location:</b> The city and country inferred from your IP address.</li>
-    <li><b>Time of request:</b> To track and manage your request.</li>
+    <li><b>IP address, approximate location and time of request:</b> For security purposes and spam prevention.</li>
```

Comment:
I honestly think this makes it harder to read instead of having each thing we collect listed on its own line. Also when flying over the PP it is more easily overlooked.